src/analyzer/protocol/dns/DNS.cc

```diff
@@ -375,9 +388,10 @@ u_char* DNS_Interpreter::ExtractName(const u_char*& data, int& len,
 		}
 
 	// Convert labels to lower case for consistency.
-	for ( u_char* np = name_start; np < name; ++np )
-		if ( isupper(*np) )
-			*np = tolower(*np);
+	if (downcase)
+        for ( u_char* np = name_start; np < name; ++np )
```

Comment:
Another place where tabs got changed to spaces

Reply:
https://github.com/zeek/zeek/pull/1015/commits/7a91b49f5a25ce91575383cd45cdd4bba971515a